cmd/pull.go

```diff
@@ -147,8 +150,11 @@ func (p *PullParams) Validate(ctx ActionCtx) error {
 	if err != nil {
 		return err
 	}
-	if oc.AccountServerURL == "" {
-		return fmt.Errorf("operator %q doesn't set account server url - unable to pull", ctx.StoreCtx().Operator.Name)
+	if oc.AccountServerURL == "" && len(oc.OperatorServiceURLs) == 0 {
```

Comment:
shouldn't be looking at OperatorServiceURLs

Reply:
see above